lib/Runtime/Library/MathLibrary.cpp

```diff
@@ -147,7 +147,7 @@ namespace Js
     {
         double result;
 
-#if defined(_M_IX86)
+#if defined(_M_IX86) && defined(_WIN32) // TODO: xplat support
```

Comment:
Yeah... this depends on whether the xplat CRT has a direct entry point to the SSE2 version of these trig functions.....  My guess is probably not :(  Even if they have it is probably under a different name...  defined(_WIN32) is probably fine.
A better comment would be great.